python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -131,6 +132,7 @@ class NCMECEntryUpdate:
     deleted: bool
     classification: t.Optional[str]
     fingerprints: t.Dict[str, str]
+    feedback: t.Optional[FEEDBACK]
```

Comment:
blocking: We should not use optional here, because empty dictionary expresses the same thing